velox/docs/functions/spark/conversion.rst

```diff
@@ -2,6 +2,29 @@
 Conversion Functions
 ====================
 
+Conversion Functions
+--------------------
+
+.. spark:function:: cast(value AS type) -> type
+
+    Explicitly cast a value as a type. As ANSI mode is OFF, an overflow will not cause an error but 
```

Comment:
When ANSI mode is disabled, explicit type casting will allow overflow values to "wrap" around rather than causing an error.